pkg/schema/tracker.go

```diff
@@ -54,9 +55,19 @@ var (
 
 // Tracker is used to track schema locally.
 type Tracker struct {
-	store kv.Storage
-	dom   *domain.Domain
-	se    session.Session
+	store     kv.Storage
+	dom       *domain.Domain
+	se        session.Session
+	toIndexes map[string]map[string]*ToIndexes
+}
+
+// ToIndexes is downstream pk/uk info.
+type ToIndexes struct {
+	schemaName string
+	tableName  string
```

Comment:
Does this used? If used, I prefer `table *filter.Table` or `tableID string` instead of `schemaName, tableName`.